lib/loader-legacy.mjs

```diff
@@ -0,0 +1,20 @@
+import { createRequire } from 'module';
+import { fileURLToPath } from 'url';
+import { send } from './ipc.mjs';
+
+const require = createRequire(import.meta.url);
+
+export async function getFormat(url, context, defaultGetFormat) {
+  const getPackageType = require('get-package-type');
+  const filePath = fileURLToPath(url);
```

Comment:
@bjornstar @kherock this crashes on any import of built in modules now, of the URL is of the type`node:<identifier>`, e.g. `node:path`.

Reply:
I created an issue for this with a little test-case: #281